app/store/slices/credentialFoyer.ts

```diff
@@ -44,17 +45,24 @@ const initialState: CredentialFoyerState = {
   pendingCredentials: [],
 };
 
+function comparableStringFor(credential: Credential): string {
+  const rawCredential = { ...credential } as Record<string, unknown>;
+
+  delete rawCredential.issuanceDate;
+  delete rawCredential.proof;
+
+  return JSON.stringify(rawCredential);
```

Comment:
One more thing here -- we can't rely on JSON.stringify() alone, since it depends on the order of the fields in the original JSON object.
That means, we'll need to [JSON-canonicalize](https://www.npmjs.com/package/json-canonicalize) both the existing VC and the incoming VC, for comparison.